### ISSUE TITLE
Accordion error

### ISSUE THREAD

**Issue by fongcindy** [GSA/px-benefit-finder#145]
Currently:
Accordion error “.usa-accordion__button[aria-controls] is missing outer .usa-accordion” when expanding

Expect:
The user should be able to open and close accordions without any error

Note from Nehemiah:
You might not experience this error in the storybook. Start benefit finder on local and past this URL and then try to open and/close the accordions

![Image](https://github.com/GSA/px-bears-drupal/assets/128498116/c9999159-ae41-4544-af0a-ea01a5a361fd)
